js/src/product-feed/issues-table-card/issues-type-navigation.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
The navigation just gets the totals per issue and updates the query on click.

Al the components subscribed to `getActiveIssueType()` like the IssuesTable will update after changing the query.